Mlem/MlemApp.swift

```diff
@@ -54,6 +55,15 @@ struct MlemApp: App {
                         appearance.backgroundEffect = UIBlurEffect.init(style: .systemThinMaterial)
                         UITabBar.appearance().standardAppearance = appearance
                         UITabBar.appearance().scrollEdgeAppearance = appearance
+                        
+                        // check whether this device has a home button
+                        if #available(iOS 11.0, *),
```

Comment:
We only support iOS 16+, right? So we don't need to check whether the user is on iOS 11+.

Reply:
Fair enough lmao, that's what I get for copying SO :P